bin/alluxio

```diff
@@ -258,6 +258,11 @@ function main {
   ;;
   "runTest")
     CLASS="alluxio.cli.TestRunner"
+    ALLUXIO_MASTER_HOSTNAME=$(${BIN}/alluxio getConf alluxio.master.hostname)
```

Comment:
`getConf` sources `libexec` which sources `alluxio-env` and sets `ALLUXIO_JAVA_OPTS+=" -Dalluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME}"`, so the value returned by `getConf` should properly factor in the environment variable